base/uk.ac.stfc.isis.ibex.client.product/pom.xml

```diff
@@ -1,6 +1,6 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-    <version>0.0.0-SNAPSHOT</version>
+    <version>3.2.0-SNAPSHOT</version>
```

Comment:
This has been branched from candidate release to ease merging onto polaris. Remember to update these version numbers back to 0.0.0 before merging to master.

(comment repeats as necessary for other files)